packages/extension-sdk-react/package.json

```diff
@@ -44,8 +44,10 @@
     "@looker/extension-sdk": "^21.4.1",
     "@looker/sdk": "^21.4.1",
     "@looker/sdk-rtl": "^21.0.12",
+    "lodash": "^4.17.20"
+  },
+  "peerDependencies": {
     "history": "^4.9.0",
```

Comment:
Does `history` need to be in `peerDependencies`? It doesn't seem like extension developers would usually need to install it alongside this package like they would the react packages.

The common pattern is to put the react packages in `devDependencies` as well, for jest, etc to work. But since this is in a monorepo where react is required in `dependencies` elsewhere, jest will still work without doing that. So I guess it would be a purely symbolic gesture. 🙂